torch/csrc/jit/mobile/backport_manager.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <functional>
+#include <memory>
+#include <unordered_map>
+
+namespace c10 {
+struct IValue;
+}
+
+namespace caffe2 {
+namespace serialize {
+class IStreamAdapter;
+class ReadAdapterInterface;
+class PyTorchStreamWriter;
+class PyTorchStreamReader;
+} // namespace serialize
+} // namespace caffe2
+
+namespace torch {
+namespace jit {
+
+constexpr const char* kArchiveNameConstants = "constants";
```

Comment:
I don't see any instance of this kArchiveNameConstants being used.

Reply:
Removed